response_operations_ui/__init__.py

```diff
@@ -26,19 +31,42 @@
 app.secret_key = app.config['RESPONSE_OPERATIONS_UI_SECRET']
 
 logger_initial_config(service_name='response-operations-ui', log_level=app.config['LOGGING_LEVEL'])
+logger = logging.getLogger(__name__)
 
 login_manager = LoginManager(app)
 login_manager.init_app(app)
 login_manager.login_view = "sign_in_bp.sign_in"
 
+if app.config['SESSION_TYPE'] == 'redis':
+    # If deploying in cloudfoundry set config to use cf redis instance
+    if cf.detected:
+        logger.info('Cloudfoundry detected, setting service configurations')
+        app.config['REDIS_HOST'] = cf.redis.credentials['host']
+        app.config['REDIS_PORT'] = cf.redis.credentials['port']
+
+    redis = redis.StrictRedis(host=app.config['REDIS_HOST'],
+                              port=app.config['REDIS_PORT'],
+                              db=app.config['REDIS_DB'])
+
+    # wrap in the flask server side session manager and back it by redis
+    app.config['SESSION_REDIS'] = redis
+
+Session(app)
+
 
 @login_manager.user_loader
 def user_loader(user_id):
-    # TODO This will need to be replaced with a call to reddis to get the token
-    # as we can't leave that sort of information in the cookie.  Will be implemented
-    # down the line once uaa is sorted out.
     return User(user_id)
 
 
+@app.after_request
```

Comment:
Can the after request be done on the info blueprint, removing the need for the info global?

Reply:
https://github.com/pallets/flask/blob/d72612d11e4f4e5d682c8986acb42ae5e8e97e4a/flask/blueprints.py#L327

Reply:
Yes good idea